src/hooks/useEngineConnectionSubscriptions.ts

```diff
@@ -209,9 +209,11 @@ export function useEngineConnectionSubscriptions() {
                     return
                   }
                   if (importDetails?.type === 'Local') {
-                    const paths = localModuleSafePathSplit(importDetails.value)
-                    const fileName = paths[paths.length - 1]
-                    showSketchOnImportToast(fileName)
+                    // importDetails has OS specific separators from the rust side!
```

Comment:
Follow up from https://github.com/KittyCAD/modeling-app/pull/6697 when Pierre and I debugged a file path issue from execstate!

Reply:
Nice, can confirm windows is working now https://kittycadworkspace.slack.com/archives/C07A80B83FS/p1746632383567239?thread_ts=1746540003.331569&cid=C07A80B83FS